addons/mail/static/src/components/chat_window_hidden_menu/chat_window_hidden_menu.js

```diff
@@ -70,6 +70,9 @@ export class ChatWindowHiddenMenu extends Component {
      * @private
      */
     _applyOffset() {
+        if (!this.root.el) {
+            return;
+        }
```

Comment:
Condition should possibly be on parent here? I would expect method not to be called when its wrong to call it.

Reply:
Same, changes done, but it's strange.